### ISSUE TITLE
[Linux] Ledger Live 2.0.0 fails to start / synchronize / access device – GLIBCXX 3.4.26

### ISSUE THREAD

**Issue by d3adb33f** [LedgerHQ/ledger-live-desktop#2670]
#### Ledger Live Version and Operating System

- tested on Ledger Live **2.0.0**
- Platform and version: **Debian 10**

#### Expected behavior

Ledger Live should be able to start and once is started should be able to synchronize and recognize  the Ledger Nano S

#### Actual behavior
Once the ledger live is executed it fails with the error message:

```
Oops, something went wrong
You may try again by restarting Ledger Live. Please export your logs and contact Ledger Support if the problem persists.
```

```
Internal process error (1)
Error
    at createCustomErrorClass (file:///tmp/.mount_ledgerjtqOhe/resources/app.asar/.webpack/renderer.bundle.js:9690:17)
    at deserializeError (file:///tmp/.mount_ledgerjtqOhe/resources/app.asar/.webpack/renderer.bundle.js:9723:25)
    at EventEmitter.handleCommandEvent (file:///tmp/.mount_ledgerjtqOhe/resources/app.asar/.webpack/renderer.bundle.js:257897:61)
    at EventEmitter.emit (events.js:210:5)
    at Object.onMessage (electron/js2c/renderer_init.js:2181:16)
```

```
redux failed: Cannot read property 'store' of null
TypeError: Cannot read property 'store' of null
    at ConnectFunction (file:///tmp/.mount_ledgerjtqOhe/resources/app.asar/.webpack/renderer.bundle.js:560:70)
    at Ih (file:///tmp/.mount_ledgerjtqOhe/resources/app.asar/.webpack/renderer.bundle.js:157129:98)
    at si (file:///tmp/.mount_ledgerjtqOhe/resources/app.asar/.webpack/renderer.bundle.js:157149:273)
    at ri (file:///tmp/.mount_ledgerjtqOhe/resources/app.asar/.webpack/renderer.bundle.js:157149:103)
    at pi (file:///tmp/.mount_ledgerjtqOhe/resources/app.asar/.webpack/renderer.bundle.js:157148:178)
    at gk (file:///tmp/.mount_ledgerjtqOhe/resources/app.asar/.webpack/renderer.bundle.js:157241:63)
    at fk (file:///tmp/.mount_ledgerjtqOhe/resources/app.asar/.webpack/renderer.bundle.js:157205:22)
    at Zj (file:///tmp/.mount_ledgerjtqOhe/resources/app.asar/.webpack/renderer.bundle.js:157204:362)
    at Lj (file:///tmp/.mount_ledgerjtqOhe/resources/app.asar/.webpack/renderer.bundle.js:157198:216)
    at file:///tmp/.mount_ledgerjtqOhe/resources/app.asar/.webpack/renderer.bundle.js:157096:115

```
 Once the application is restarted (as per one of the the options presented in the error message) ledger live is not able to able to recognize the Nano S and is not able to synchronize with the network, it keeps in the "Synchronizing..." state forever.   

#### Steps to reproduce the behavior

1- Download AppImage
2- Launch ledger live: ./ledger-live-desktop-2.0.0-linux-x86_64.AppImage 

Previous version was working fine...

**Comment by tomvlk** [comment]
I have the same issue here.

```
Internal process error (1)
Error
    at createCustomErrorClass (file:///tmp/.mount_ledger2KtQhm/resources/app.asar/.webpack/renderer.bundle.js:9690:17)
    at deserializeError (file:///tmp/.mount_ledger2KtQhm/resources/app.asar/.webpack/renderer.bundle.js:9723:25)
    at EventEmitter.handleCommandEvent (file:///tmp/.mount_ledger2KtQhm/resources/app.asar/.webpack/renderer.bundle.js:257897:61)
    at EventEmitter.emit (events.js:210:5)
    at Object.onMessage (electron/js2c/renderer_init.js:2181:16)
```

```
redux failed: Cannot read property 'store' of null
TypeError: Cannot read property 'store' of null
    at ConnectFunction (file:///tmp/.mount_ledger2KtQhm/resources/app.asar/.webpack/renderer.bundle.js:560:70)
    at Ih (file:///tmp/.mount_ledger2KtQhm/resources/app.asar/.webpack/renderer.bundle.js:157129:98)
    at si (file:///tmp/.mount_ledger2KtQhm/resources/app.asar/.webpack/renderer.bundle.js:157149:273)
    at ri (file:///tmp/.mount_ledger2KtQhm/resources/app.asar/.webpack/renderer.bundle.js:157149:103)
    at pi (file:///tmp/.mount_ledger2KtQhm/resources/app.asar/.webpack/renderer.bundle.js:157148:178)
    at gk (file:///tmp/.mount_ledger2KtQhm/resources/app.asar/.webpack/renderer.bundle.js:157241:63)
    at fk (file:///tmp/.mount_ledger2KtQhm/resources/app.asar/.webpack/renderer.bundle.js:157205:22)
    at Zj (file:///tmp/.mount_ledger2KtQhm/resources/app.asar/.webpack/renderer.bundle.js:157204:362)
    at Lj (file:///tmp/.mount_ledger2KtQhm/resources/app.asar/.webpack/renderer.bundle.js:157198:216)
    at file:///tmp/.mount_ledger2KtQhm/resources/app.asar/.webpack/renderer.bundle.js:157096:115
```

**Comment by sanchopansa** [comment]
Same issue for me as well:

```
Internal process error (1)
Error
    at createCustomErrorClass (file:///tmp/.mount_ledgerTMqsav/resources/app.asar/.webpack/renderer.bundle.js:9690:17)
    at deserializeError (file:///tmp/.mount_ledgerTMqsav/resources/app.asar/.webpack/renderer.bundle.js:9723:25)
    at EventEmitter.handleCommandEvent (file:///tmp/.mount_ledgerTMqsav/resources/app.asar/.webpack/renderer.bundle.js:257897:61)
    at EventEmitter.emit (events.js:210:5)
    at Object.onMessage (electron/js2c/renderer_init.js:2181:16)
```

**Comment by gre** [comment]
Ok thanks for the log. We seem to have an issue on Ubuntu 18.04. We'll provide a fix as soon as possible.

**Comment by d3adb33f** [comment]
Not only on Ubuntu but apparently on any Debian based distro.

**Comment by 2hd3x** [comment]
Same problem on Linux Mint 19

**Comment by gre** [comment]
It is essentially related to the minimal glibc version. You might want to try to `apt-get dist-upgrade` (depending on distrib). We still plan to fix this. If it still doesn't work, you can stay on 1.20.0 (and don't apply the update)

**Comment by zsoltsandor** [comment]
Basically, the same issue. Ubuntu 18.04, and all that jazz.
Finally, it started working on my tablet, now it's failing on my laptop. :(

**Comment by 13albra** [comment]
where i can find the previous version ledger live windows 10 64bit...2.0.0 not working....need ugrent my crypto

**Comment by 13albra** [comment]
![Untitled](https://user-images.githubusercontent.com/61806005/75931751-8dde1280-5e2a-11ea-87f7-d049efbf1519.jpg)

**Comment by cryptoTaxes** [comment]
Same problems on Debian 9. FYI, latest GLIBCXX on Debian 9 is 3.4.22

**Comment by JamesSeeley27** [comment]
Will Ledger notify each of us by email, or return to Github for further updates for us linux users?

**Comment by gre** [comment]
@JamesSeeley27 we'll keep this task up to date and if we have a fix 🤞, a release notes will be written on a release.

**Comment by RickSls** [comment]
The same issue on Debian 10 
[5920:0305/105052.026353:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /tmp/.mount_ledgerjGA57m/chrome-sandbox is owned by root and has mode 4755.
Trace/breakpoint trap

**Comment by tomvlk** [comment]
@13albra and all, you can download the old version here: https://github.com/LedgerHQ/ledger-live-desktop/releases/tag/v1.20.0

**Comment by F4brtB4425** [comment]
Hi guys. I have exactly the same problem after upgrading to Ledger Live V2.0 in Windows 10!!!! Can someone please let me know how to solve this problem? Many thanks to all.

**Comment by MortalKastor** [comment]
The source of the issue on Linux has been pin-pointed and a fix is on the way.

On Windows, the investigation is still ongoing but we found a bug which most probably is causing the crash, we're testing a fix right now to confirm.

**Comment by gre** [comment]
As both problem are independent, for windows problem, we are going to use this different github task https://github.com/LedgerHQ/ledger-live-desktop/issues/2680 

We are still investigating what could be the root cause of the Windows issue and have not found yet.

For linux, we will have a fix and build a new release as soon as possible.

**Comment by F4brtB4425** [comment]
Hi

Here is the Log File from yesterday. Anxiously awaiting your eresponse.

Thanks very much

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by Gundelino85** [comment]
> As both problem are independent, for windows problem, we are going to use this different github task #2680
> 
> We are still investigating what could be the root cause of the Windows issue and have not found yet.
> 
> For linux, we will have a fix and build a new release as soon as possible.

What do you think when "as soon as possible" will be? Cant use my nano s right now...

**Comment by michnovka** [comment]
> What do you think when "as soon as possible" will be? Cant use my nano s right now...

Yes you can. Scroll up, find link to latest previous stable version. Download. Use.

https://github.com/LedgerHQ/ledger-live-desktop/releases/tag/v1.20.0

**Comment by Gundelino85** [comment]
> > What do you think when "as soon as possible" will be? Cant use my nano s right now...
> 
> Yes you can. Scroll up, find link to latest previous stable version. Download. Use.
> 
> https://github.com/LedgerHQ/ledger-live-desktop/releases/tag/v1.20.0

No i can't. I get this error: "Failed to load module "canberra-gtk-module""

**Comment by Gundelino85** [comment]
No, i can't. Get this error: "Cannot find module '../dialog'"

**Comment by Gundelino85** [comment]
Works after deleting "/home/USER/.config/Ledger\ Live" created by 2.0.0

**Comment by Gundelino85** [comment]
Version 2.0.1 on Ubuntu: "Device detected but connection failed. Please try again or contact us if the problem persists."

**Comment by gre** [comment]
The issue **on Linux** should now be fixed by an incoming 2.0.1

It is currently in prerelease https://github.com/LedgerHQ/ledger-live-desktop/releases/tag/v2.0.1

and should be released on Monday if we confirm all is fine.

Thanks for everyone helping us in this GitHub issue.

**Comment by Gundelino85** [comment]
Have you read my message @gre ?

**Comment by RickSls** [comment]
The issue persists on Debian 10
**$ ~/Apps/ledger-live-desktop-2.0.1-linux-x86_64.AppImage 
[**19211:0305/191014.276182:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /tmp/.mount_ledgertS4auA/chrome-sandbox is owned by root and has mode 4755.
Trace/breakpoint trap

**Comment by gre** [comment]
@Gundelino85 totally missed it sorry. Can you create a new issue? it seems unrelated to the first issue we have fixed with GLIBCXX
we will need to investigate.

**Comment by F4brtB4425** [comment]
Hi guys

I've been sitting in front of my PC since last night sort of freaking out that my funds were lost or I got hacked or WTF is going on. I even reset my Ledger Nano S but I just got myself in deeper trouble. I reinstalled V1.20.0 like you suggested it to the guy that has the same problem on Linux, and I did a Seed Recovery. Now everything is back to normal. Got my accounts and funds restored!!!!!!!  I'm familiar with PCs but new to Crypto. This was a massive learning curve emotionally and of course technically as well!!! :D The next time no way I'm doing an update as soon as it's available. Gonna wait it out and see how others will do with the newer versions.

Thanks very much for the quick replies much appreciated. I assume regardless of me solving my problem there will be an update to fix this. Can you please keep me in the loop of communication regarding this in the future.

Thanks again guys gonna have a drink for sure now (Y) :D

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by nix4me** [comment]
I also have had the same issues in linux (slackware) as above with an added Fontconfig error that nobody else seems to have. After updating to v2,0,0 and closing ledger live it would not startup again until I renamed ~/.config/Ledger Live/ and letting it create a new Ledger Live directory but failed to connect to my nano s. The Fonconfig error was:

$./ledger-live-desktop-2.0.0-linux-x86_64.AppImage 
Fontconfig warning: "/etc/fonts/fonts.conf", line 102: unknown element "blank"
Fontconfig warning: "/etc/fonts/conf.avail/66-wqy-zenhei-sharp-no13px.conf", line 11: Having multiple values in <test> isn't supported and may not work as expected
Fontconfig warning: "/etc/fonts/conf.avail/66-wqy-zenhei-sharp.conf", line 10: Having multiple values in <test> isn't supported and may not work as expected
Ledger Live 2.0.0

So because I updated to v2.0.0 through the app I couldn't use the original ~/.config/Ledger Live
so I went back to v1.20.0 and let it create the Ledger Live dir. and installed all the apps and it works fine again.

Trying v2.0.1 it also fails unless I start without the ~/.config/Ledger Live directory spewing the same Fontconfig error as above.
